dspace-api/src/main/java/org/dspace/content/dao/impl/MetadataFieldDAOImpl.java

```diff
@@ -77,7 +80,7 @@ public MetadataField findByElement(Context context, String metadataSchema, Strin
         throws SQLException {
         Query query;
 
-        if (StringUtils.isNotBlank(qualifier)) {
+        if (qualifier != null) {
```

Comment:
Why are we removing `StringUtils.isNotBlank()` in favor of `!=`?  These aren't the same thing.